MetaMask/metamask-extension
# Feat/collectibles display

Explanation:  Adds remaining required wiring + front-end work to get NFTs MVP in extension

In designs but excluded here
- Send flow (I think we agreed somewhere that this wasn't required for MVP... but I might totally be making this up)
- `Report as Scam` flow (the functionality here is not sufficiently spec'd out to implement)

Screenshots:
<img width="684" alt="Screen Shot 2021-11-30 at 4 09 50 PM" src="https://user-images.githubusercontent.com/34557516/144136219-ae84e63d-822e-4a94-98c6-420eac365a38.png">
<img width="944" alt="Screen Shot 2021-11-30 at 4 10 54 PM" src="https://user-images.githubusercontent.com/34557516/144136279-aa553a70-ebf9-48b6-a445-b2877e947a93.png">

Manual testing steps:  
  - Purchase or mint or have an (or preferably many) NFT(s)
  Detection:
  - Turn on NFT autodetection in settings -> experimental and wait 2 mins
  - Check that your NFT has been added to the NFTs tab
  Manual:
  - Navigate to the NFTs tab
  - Click "Import NFTs"
  - Check that your NFT has been successfully added
